src/custom-elements/src/components/toggle-switch/toggle-switch.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component, Event, EventEmitter, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'chi-toggle-switch',
+  styleUrl: 'toggle-switch.scss',
+  scoped: true
+})
+export class ToggleSwitch {
+  /**
+   *  to set the toggle switch id.
+   */
+  @Prop({ reflectToAttr: true }) toggleId: string;
+
+  /**
+   *  to set a label for the toggle switch.
+   */
+  @Prop({ reflectToAttr: true }) label: string;
+
+  /**
+   *  used to disable the toggle labels.
+   */
+  @Prop({ reflectToAttr: true }) noText = false;
+
+  /**
+   *  used to disable the toggle switch.
+   */
+  @Prop({ reflectToAttr: true }) disabled = false;
+
+  /**
+   *  used to check the toggle switch.
+   */
+  @Prop({ reflectToAttr: true, mutable: true }) checked = false;
+
+   // used to pass additional classes like { -focus, -hover etc .}
```

Comment:
Just for consistency sake, could you please set this comment like this? 
```suggestion
   /**
   *  used to pass additional classes like { -focus, -hover etc .}
   */
```

Reply:
Ok. Just wondering..would it be possible to pass several classes at the same time? Would it make sense an array of classes?

Reply:
I agree. It sounds more 'clean' than several classes within a string

Reply:
Yes, you're right. 
Leave it like that but...how about renaming it to extraClasses?

Reply:
Ok, let's not reinvent the wheel :)

I agree with that. Thank you.